### ISSUE TITLE
Oracle dialect handles timestamps inconsistently

### ISSUE THREAD

**Issue by snehlsen** [exasol/virtual-schemas#99]
The oracle dialect handles timestamps differently depending on if you use a projection or not:
```sql
select * from docker_oravs.tstest; 
-->
TS                   TSTZ                TSLTZ
2013-09-11 12:55:05	2013-09-11 12:55:05	2013-09-11 19:55:05
```
```sql
select TS from docker_oravs.tstest; 
-->
TS                                  
11-SEP-13 12.55.05.123456 PM
```
The reason is that the projection adds `TO_CHAR` for timestamps and hence applies the `NLS_TIMESTAMP_FORMAT`.

**Comment by snehlsen** [comment]
@andrehacker was there any reason for this or is it just a bug?

**Comment by snehlsen** [comment]
It looks like for an ORA connection you don't have to cast TO_CHAR, but for a JDBC connection you have to. In order to fix this the dialect would have to know about the type of connection that is used. This currently not possible and addressed in #96

**Comment by andrehacker** [comment]
@snehlsen I can't remember anything about this, probably I didn't write this dialect.
But seems you already figured something out. What you write, also in #96 makes sense.

**Comment by snehlsen** [comment]
Fixed in version 1.8.1
